Orange/widgets/data/owpaintdata.py

```diff
@@ -1223,6 +1237,10 @@ def pen(color):
             pen.setCosmetic(True)
             return pen
 
+        def brush(color):
```

Comment:
I suppose this function is here only because `pen` has its own method. Otherwise this is not needed.